index.html

```diff
@@ -10,30 +10,71 @@
   <script src="java.js" defer></script>
 </head>
 <body class="mm">
-  <h1>Awesome Books</h1>
-  <table class="tables">
-    <!--<thead>
-      <tr>
-        <th>Title</th>
-        <th>Author</th>
-        <th></th>
-      </tr>
-    </thead> <br>-->
-    <tbody id="bookli"></tbody>
-  </table>
-  <hr>
-  <form id="bookform" class="book-form">
-    <h3 class="titlemain txt-center">Add a new book</h3>
-    <div class="forms mbb">
-      <input type="text" placeholder="Title" id="title" class="title-input" required>
-    </div>
-    <div class="forms">
-      <input type="text" placeholder="Author" id="author" class="author-input" required>
-    </div>
-    <div class="forms">
-      <input type="submit" value="Add"  class="button-submit">
-    </div>
-  </form>
+  <header>
+    <nav>
+        <a href="./index.html" class="logo">Awesome Books</a>
+        <ul>
+            <li id = "list">
+                <a href="#list">List</a>
+            </li>
+            <li id="addnew">
+                <a href="#addnew">Add new</a>
+            </li>
+            <li id="contac">
+                <a href="#contact">Contact</a>
+            </li>
+        </ul>
+    </nav>
+    <h3 id ='time'>
+    </h3>
+
+  </header>
+  <main>
+    <section id="top">
+      <h2>
+        All awesome books
+      </h2>
+      <div class="all"></div>
+    </section>
+    <table class="tables">
+      <!--<thead>
+        <tr>
+          <th>Title</th>
+          <th>Author</th>
+          <th></th>
+        </tr>
+      </thead> <br>-->
+      <tbody id="bookli"></tbody>
+    </table>
```

Comment:
- [ ] Kindly note that the list of books should only appear when the `list` link in the navbar is clicked. currently, I can see the list of books on all pages. kindly remove the list of books on the contact and form pages

![image](https://user-images.githubusercontent.com/65068771/212442296-9e1a38fb-81d4-4657-b351-7bf914e1c92e.png)
 Check this as an :point_right: [example](https://andersonlebon.github.io/Awesome-book/)